src/worker/views/playerBios.ts

```diff
@@ -24,13 +25,36 @@ const updatePlayers = async (
 			hockey: ["keyStats"],
 		});
 
+		const config: TableConfig = new TableConfig("playerBios", [
+			"Name",
+			"Pos",
+			"stat:jerseyNumber",
+			"Team",
+			"Age",
+			"Height",
+			"Weight",
+			"Mood",
+			"Contract",
+			"Exp",
+			"Country",
+			"College",
+			"Draft Year",
+			"Pick",
+			"Experience",
+			"Ovr",
+			"Pot",
+			...stats.map(s => `stat:${s}`),
+		]);
+		await config.load();
+
 		const players = await getPlayers(
 			inputs.season,
 			inputs.abbrev,
-			["born", "college", "hgt", "weight", "draft", "experience"],
-			["ovr", "pot"],
-			[...stats, "jerseyNumber"],
+			[],
+			[],
+			[],
```

Comment:
How is this supposed to work... where are the things deleted from here going to come from?

Reply:
At somepoint i was going to merge playerBios, playerStats, and playerRatings but decided that was too radical. fixed this